### PR TITLE
General website updates

### LINKED ISSUE #58: Update website style
The website seems to be outdated and should be updated to use the latest version of bootstrap and Epiverse theme.

### LINKED ISSUE #65: Move quickstart in README to website
The current quickstart section in the README is probably best placed on the package website. 

The quick start in the README should be simple and point out the current functionality without providing too many details on the inner workings of the functions.

### LINKED ISSUE #70: Organize vignettes into topics
I propose to restructure the website vignettes into the following topics:
     - [ ] Applications of branching processes (bibliography of modelling literature that uses branching processes)
     - [ ] Modelling (including forecasting, etc.)

### PR DESCRIPTION
**Pull request by jamesmbaazam**
This PR closes #58, fixes #65, and closes #70 by:

- Moving the quickstart section of the README to the website
- Cleaning up the README to have a minimalistic look.
- Updating the website style to use Bootstrap 5 and matching the style of the other Epiverse packages.
- Minor changes in function docs/titles to aid website cleanup.

### CODE REVIEW

File: vignettes/bpmodels.Rmd [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+title: "Analysing infectious disease transmission chains"
```
Comment (sbfnk) [inline vignettes/bpmodels.Rmd:2]: Maybe
```suggestion
title: "Analysing infectious disease transmission chains with `bpmodels`"
```
?

File: README.Rmd [hunk 2]
```diff
@@ -47,133 +49,21 @@ To load the package, use
...
-be interpreted as the reproduction number $\mathcal{R_0}$) of $0.5$. 
-
-To do this, we run 
-
-```{r}
-set.seed(13)
-chain_sizes <- c(1, 1, 4, 7) # example of observed chain sizes
-chain_ll(x = chain_sizes, offspring = "pois", stat = "size", lambda = 0.5)
-```
+_bpmodels_ provides three functions:
```
Comment (sbfnk) [inline README.Rmd:54]:
```suggestion
_bpmodels_ provides three main functions: 
```

**Review by sbfnk — APPROVED** [review]
LGTM

File: vignettes/bpmodels.Rmd (hunk 1 above)
Comment (jamesmbaazam) [reply vignettes/bpmodels.Rmd:2]: I like it.

File: README.Rmd [hunk 3]
```diff
@@ -47,133 +49,21 @@ To load the package, use
...
-serial_interval <- function(n) {
-  rlnorm(n, meanlog = 0.58, sdlog = 1.58)
-}
-chains_df <- chain_sim(
-  n = 5, offspring = "pois", lambda = 0.5, stat = "length",
-  infinite = 100, serial = serial_interval, tree = TRUE
-)
-head(chains_df)
-```
+See the ["introduction vignette"](bpmodels) for a detailed illustration of
```
Comment (avallecam) [inline README.Rmd:65]: Looks good to me. It communicates directly the core actions of the package's main functions and gives more space to the "Get started" vignette. 
Some details: 
- this link is not working for me locally. Probably `(articles/bpmodels)` works? I'm rendering it with `pkgdown::build_site()`
- Replacing `"introduction vignette"` with `"Get started" vignette` could indicate clearly to the content in the navbar called "Get started". Possibly too obvious, but it makes me think if there is another intro vignette different to the get started.

With these comments, I can close my review. I'll give further feedback on the "Get started" vignette content in separate issues.

**Review by avallecam — CHANGES_REQUESTED** [review]
request to review the link redirecting to the "Get started" vignette

File: README.Rmd (hunk 3 above)
Comment (jamesmbaazam) [reply README.Rmd]: Thanks, Andree. I have fixed that in 23f44e8b69f7b4d9d672e1e21707ef287797b1ca.

**Pull request merged**